terraform/vars/tvm-ci-prod.auto.tfvars

```diff
@@ -119,6 +119,15 @@ autoscaler_types = {
     on_demand_percentage_above_base_capacity = 0
     on_demand_base_capacity                  = 0
   }
+  "Prod-Autoscaler-Jenkins-ARM-Graviton3" = {
+    image_family                             = "jenkins-stock-agent-arm"
+    agent_instance_type                      = "c7g.large"
+    labels                                   = "ARM-GRAVITON3"
```

Comment:
Can you also add an `ARM-GRAVITON3-SPOT` type as well that follows the pattern of the others? For the CI we've transitioned to using AWS Spot runners to save costs, falling back to on-demand only when those get terminated during a job.